ramls/inventory/loccamp.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "second-level location unit",
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "name": {
+      "description": "name of the location",
+      "type": "string"
+    },
+    "code": {
+      "description": "distinct code for the location",
+      "type": "string"
+    },
+    "institutionId": {
+      "description": "ID of the first-level location unit that the second-level unit belongs to",
+      "type": "string"
+    },
+    "metadata": {
+      "type": "object",
+      "$ref": "../raml-util/schemas/metadata.schema",
+      "readonly": true
+    }
+  },
+  "additionalProperties": true,
+  "required": [
+    "name", "institutionId", "code"
```

Comment:
maybe it makes sense to to split required fields into multiple lines, one field for one line to be in the same style like in other schemas? or conversely put them on one line for all schemas?